### ISSUE TITLE
Cant serialize string[]

### ISSUE THREAD

**Issue by cryptiklemur** [danrevah/typeserializer#5]
**Describe the bug**
Can't serialize an object that has a `string[]` property

**To Reproduce**
```typescript
import {ExclusionPolicy, Expose, serialize, Strategy} from 'typeserializer';

@Strategy(ExclusionPolicy.NONE)
class Test {
    @Expose()
    roles: string[] = ['ROLE_USER', 'ROLE_ADMIN'];
}

console.log(serialize(new Test()));
```

**Expected behavior**
Expect to see `{"roles": ["ROLE_USER", "ROLE_ADMIN"]}`

**Desktop (please complete the following information):**
 - OS: macOS
 - Node version: v9.11.2
 - Npm version: 5.6.0
 - Typescript version: 2.9.2
 - TypeSerializer Version: 0.1.9

**Comment by cryptiklemur** [comment]
The error i get:

```
TypeError
    at Object.getMetadata (/Users/aaron/Projects/javascript/discordservers/api/node_modules/reflect-metadata/Reflect.ts:994:42)
    at transform (/Users/aaron/Projects/javascript/discordservers/api/node_modules/src/Serializer/Serializer.ts:31:30)
    at arr.map (/Users/aaron/Projects/javascript/discordservers/api/node_modules/src/Serializer/Serializer.ts:26:79)
    at Array.map (<anonymous>)
    at transformArray (/Users/aaron/Projects/javascript/discordservers/api/node_modules/src/Serializer/Serializer.ts:25:14)
    at Object.getOwnPropertyNames.reduce (/Users/aaron/Projects/javascript/discordservers/api/node_modules/src/Serializer/Serializer.ts:55:22)
    at Array.reduce (<anonymous>)
    at transform (/Users/aaron/Projects/javascript/discordservers/api/node_modules/src/Serializer/Serializer.ts:48:42)
    at Object.serialize (/Users/aaron/Projects/javascript/discordservers/api/node_modules/src/Serializer/Serializer.ts:20:25)
    at Object.<anonymous> (/Users/aaron/Projects/javascript/discordservers/api/src/main.ts:15:13)
```

**Comment by danrevah** [comment]
Thanks for letting me know that it's failing.

I've just released a fix to version `v0.2.1`.

Please note that you are using an exclusion strategy of 'ExclusionPolicy.NONE', and attempting to `Expose()` properties.
It will expose every property anyway, unless you use `Exclude()`.

If you want to manually `Expose` use `ExclusionPolicy.ALL`.
